contract/src/msgs/staking/tests.rs

```diff
@@ -279,6 +279,46 @@ fn multiple_executor_eligible_exact_replication_factor() {
     assert!(is_val1_executor_eligible && is_val2_executor_eligible);
 }
 
+#[test]
+fn only_allow_active_stakers_to_be_eligible() {
```

Comment:
Would a better test not be to use a single executor and first register but with too little stake, see it's not eligible, up the stake to match the minimum, and verify it's now eligible?